core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3129,13 +3146,34 @@ init_llvm_jit_functions_stage2(WASMModule *module, char *error_buf,
     }
 
     bh_print_time("End lookup llvm jit functions");
+#endif
 
+    (void)i;
     return true;
 }
 #endif /* end of WASM_ENABLE_JIT != 0 */
 
 #if WASM_ENABLE_FAST_JIT != 0 && WASM_ENABLE_JIT != 0 \
     && WASM_ENABLE_LAZY_JIT != 0
+
+static bool
```

Comment:
Had better wrap this function with `#if WASM_ENABLE_DYNAMIC_PGO != 0  ... #endif`